pkg/utils/fetcher/fetchEtcd.go

```diff
@@ -0,0 +1,85 @@
+package fetcher
```

Comment:
Looks like this "fetcher" can only fetch cluster info. So why not call it clusterinfo?

Reply:
emmm, well, I'll change it

Reply:
Done, PTAL